src/BackTranslationHelper/BackTranslationHelperCtrl.cs

```diff
@@ -268,54 +306,55 @@ public DirectableEncConverter QueryTranslator()
             return theEc;
         }
 
-        public List<TargetPossible> NewTargetTexts
-        {
-            get
-            {
-                // always choose the one in the text box
-                var newTargetTexts = new List<TargetPossible> 
-                { 
-                    new TargetPossible 
-                    { 
-                        TargetData = textBoxTargetBackTranslation.Text,
-                        PossibleIndex = 0,
-                        TranslatorName = TheTranslators.FirstOrDefault()?.Name
-                    } 
-                };
-                return newTargetTexts;
-            }
-
-            set
-            {
-				var textBoxesPossibleTargetTranslations = tableLayoutPanel.Controls.OfType<TextBox>().Where(l => l.Name.Contains("textBoxPossibleTargetTranslation")).ToList();
-				System.Diagnostics.Debug.Assert(value.Count == TheTranslators.Count);
-				System.Diagnostics.Debug.Assert(textBoxesPossibleTargetTranslations.Where(l => l.Visible).Take(value.Count).ToList().All(l => l.Visible));
-
-				for (var i = 0; i < TheTranslators.Count; i++)
+		public List<TargetPossible> GetNewTargetTexts()
```

Comment:
This file is now a mix of tab vs space indentation, that could be addressed in a future commit.

Reply:
I think this is fixed... VS 2022 doesn‘t seem to like to use spaces for tabs. I have it configured to use spaces and disable the Advance, Use adaptive formatting, and it still puts in a \t if I press the tab key...